release/release_tests.yaml

```diff
@@ -3132,6 +3132,26 @@
     type: sdk_command
     file_manager: sdk
 
+- name: dataset_random_access
+  group: core-dataset-tests
+  working_dir: nightly_tests/dataset
+
+  frequency: multi
+  team: core
+  cluster:
+    cluster_env: pipelined_training_app.yaml
+    cluster_compute: pipelined_training_compute.yaml
+
+  run:
+    timeout: 1200
+    script: python dataset_random_access.py
+    wait_for_nodes:
+      num_nodes: 15
+      timeout: 1200
+
+    type: sdk_command
+    file_manager: sdk
```

Comment:
@rkooo567 did I get this config right?

Reply:
Yes! Do you need to mark it as stable:false btw?